tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -172,8 +189,8 @@ public function validInputProvider() {
 				$gregorian,
 				$gregorianOpts,
 			),
-			'-1-01-03T00:00:00Z' => array(
-				'-0000000000000001-01-03T00:00:00Z',
+			'+2001-01-03T00:00:00Z' => array(
+				'+0000000000002001-01-03T00:00:00Z',
```

Comment:
Because otherwise, the test would be redundant. We are trying to test that the Julian calendar in the options takes precedence over the "guessed" year. So the input has to be a year that does not default to Julian.